ourstory.html

```diff
@@ -15,7 +16,7 @@ <h1 style="background-color:green;">Our Story</h1>
         Buffalo in 1950. The brothers opened up <em>"Pizzeria di Pardi"</em> on Grant street in Buffalo's<br/>
         west side. In 1986 we expanded to 8 locations in western NY and Pizzeria di Pardi became<br/>
         <b>Pizza Party!</b> Today we have three locations serving up the classic pizza & wings Buffalo <br/>
-        is known for our original location as well as our Hertel Aveneue and UB locations.
+        is known for at our original location as well as our Hertel Aveneue and UB locations.
     </p>
     <img src="/Users/christopherlainson/Desktop/workspace/PizzaParty/img/party_boys.jpg" Alt="Pardi Bros Havana NYE 1958 colorized" style="width:560px;height:325px"><!-- move these styles to css -->
```

Comment:
this path is wrong. It only refers to your local machine it needs to be the [relative path](https://desktop.arcgis.com/en/arcmap/10.3/tools/supplement/pathnames-explained-absolute-relative-unc-and-url.htm#GUID-5118AC85-57E4-4027-AC24-FB6E99FADEFF)

Reply:
image path corrected